### ISSUE TITLE
Update existing events to use new sponsor format

### ISSUE THREAD

**Issue by mattstratton** [devopsdays/devopsdays-web#123]
We have determined it is best to do the shared sponsors on a per-year basis (to help with sponsor logos changing over time). We need a volunteer to go through the existing `/data/events` folder and update all the sponsors to be named accordingly. Additionally, the existing `/data/sponsors` files need to be renamed to match these names, and the `/static/img/sponsors` directory files need to be renamed. 

It is up to the person volunteering to do this to decided between `2016-chef` or `chef-2016` as the standard; just pick one, and whatever you decide, update the README as well (currently the README says to use `chef-2016`).

**Comment by mattstratton** [comment]
No longer needed. We will eventually handle this in the future with #125 but it's not needed in 2016.

**Comment by mattstratton** [comment]
This depends on #133, just for reference.
